qds_sdk/cluster.py

```diff
@@ -1037,7 +1043,7 @@ def set_cluster_info(self, aws_access_key_id=None,
         self.enable_ganglia_monitoring = enable_ganglia_monitoring
         self.node_bootstrap_file = node_bootstrap_file
         self.__set_node_configuration(master_instance_type, slave_instance_type, initial_nodes, max_nodes, slave_request_type, fallback_to_ondemand)
-        self.__set_ec2_settings(aws_access_key_id, aws_secret_access_key, aws_region, aws_availability_zone, vpc_id, subnet_id, role_instance_profile)
+        self.__set_ec2_settings(aws_access_key_id, aws_secret_access_key, aws_region, aws_availability_zone, vpc_id, subnet_id, master_elastic_ip, role_instance_profile)
```

Comment:
Please write a brief description of what master_elastic_ip parameter does in the comments section written just after the start of this function.